src/main/resources/api-specs/security-openapi.yaml

```diff
@@ -80,6 +79,34 @@ paths:
               schema:
                 $ref: "#/components/schemas/UserAuthenticationResponse"
 
+  /api/v1/auth/logout:
+    post:
+      tags:
+        - "Security"
+      summary: "Logout user"
+      operationId: "logout"
+      responses:
+        "200":
+          description: "OK"
+
+  /api/v1/auth/refresh:
```

Comment:
Could you please check what't wrong with Swagger? It generates incorrect paths:
<img width="599" alt="Screenshot 2024-01-22 at 20 48 09" src="https://github.com/Sunagatov/Iced-Latte/assets/20650874/449856e4-e29b-411e-a10d-fb88386a69ad">

To get access to Swagger please temporary remove `src/main/java/com/zufar/icedlatte/ApplicationMigration.java`